js/components/tabs.js

```diff
@@ -62,21 +76,41 @@ var Tabs = React.createClass({
       .map(function(panel, index)  {
         var ref = ("tab-menu-" + (index + 1));
         var title = panel.props.title;
+        var activeMenuColor =  "solid ";
+        activeMenuColor += this.props.skinConfig.controlBar.iconStyle.active.color ? 
+                           this.props.skinConfig.controlBar.iconStyle.active.color : 
+                           this.props.skinConfig.general.accentColor;
+
+        var activeTabStyle = { borderBottom: activeMenuColor};
+
         var classes = ClassNames(
           'tabs-menu-item',
           this.state.tabActive === (index + 1) && 'is-active',
           'tabs-menu-item-' + index
         );
 
+      if ( classes.search("is-active") === -1 ) {
```

Comment:
please simplify this conditional statement to determine `activeTabStyle` instead of duplicating entire list-item.

also use logic that determines 'is-active' instead of searching for className